config/initializers/auth0.rb

```diff
@@ -0,0 +1,12 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider(
+    :auth0,
+    'JQ5uoA8wiuSsZ5Z34oddLSGb2dfn2sJ6',
```

Comment:
We can change this to `ENV['AUTH0_CLIENT_ID']` I suppose

Reply:
Yeah, you're dead right. I was having loads of issues with secrets / credentials / env as their docs said to do this a different way. This was me being lazy / troubleshooting. Will update!